owncloud/core
# add method to count users by loginname and use it for diagnose test, …

…fixes #20151 

Test steps:
1. Configure an LDAP connection until finishing Login Filter tab
2. Test a valid Login name

Before: Instead of a success message "UserMapper was not assigned to this Access instance"  is shown
Now: Works

Please test and review @Xenopathic @PVince81 @GreenArchon @davitol 

@karlitschek needs to be backported til 8.1 – ok?
